tasks/notify.py

```diff
@@ -140,10 +140,14 @@ def failure_summary_send_notifications(
 
 
 @task
-def unit_tests(ctx, pipeline_id, pipeline_url, branch_name):
+def unit_tests(ctx, pipeline_id, pipeline_url, branch_name, print_to_stdout=False):
```

Comment:
```suggestion
def unit_tests(ctx, pipeline_id, pipeline_url, branch_name, dry_run=False):
```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I find the naming a bit confusing, imo what you added is a `--dry-run` flag. As a user if I see a `--print-to-stdout` flag I would expect the msg to be printed and then the pr comment to be made. WDYT ?